packages/core/admin/admin/src/hooks/useFetchMarketplacePlugins/utils/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+const MARKETPLACE_API_URL = 'https://market-api.strapi.io';
+
+const fetchMarketplacePlugins = async notify => {
+  const { data: response } = await axios.get(`${MARKETPLACE_API_URL}/plugins`);
```

Comment:
Is there a reason why you aren't using `axiosInstance` here?

Reply:
The axios instance is for making requests to the Strapi backend base url. Here are making requests to the marketplace API proxy. I guess we could setup another axios instance but for this iteration, in my opinion, this is fine.

Reply:
It's fine to just use axios here the instance is just made to add specific headers for requesting the admin API I don't think it's needed to create a new one.